docs/io.cozy.bills.md

```diff
@@ -16,6 +16,12 @@
 
 ### Optional attributes (but some are important depending the context)
 
+- `recurring`: {Boolean|String} : is the bill a recurring one ?
+    - `Boolean:true`: the bill is recurring every month in the day corresponding to the `date` attribute.
+      Ex: date: 28/12/2019, recurring: true -> cron expression 0 0 0 28 * *
+          date: 31/12/2019, recurring: true -> cron expression 0 0 0 28-31 * * (but the service
+          will need to find a clever way to have only one recurring operation by month.
+    - `String`: a cron expression to get all th scheduling possibilities
```

Comment:
I am not a fan of the Boolean true attribute that supports only monthly recurrence.
Could we remove the boolean value and have only strings, with potentially built-ins ?

recurring: "monthly"
recurring: "bi-monthly"
recurring: "0 0 0 28 * *"

Reply:
Ok : monthly, bi-monthly, yearly, weekly